support/routes/v2/isobot/index.ts

```diff
@@ -45,3 +78,112 @@ bot.command("/siteup", async ({ payload, respond, ack }) => {
 
   return botService.dnsChecker(payload).map((response) => respond(response))
 })
+
+bot.command(
+  "/clone",
+  validateIsomerUser,
+  async ({ command, ack, respond, payload, client }) => {
+    await ack()
+
+    const HELP_TEXT =
+      "Usage: `/clone <github_repo_name>`. Take note that this locks the repo for 15 minutes by default. To bypass this behaviour, add `-n` at the end of the command"
```

Comment:
us clone the right term? we usually refer to it as a repair?

Reply:
the repair does a `lock -> clone -> unlock` under the hood!

Reply:
correct, hence the 3-step process is called `repair`